app/build.gradle

```diff
@@ -24,13 +24,18 @@ android {
         buildConfigField 'String', 'LIGHTNING_VERSION_NAME', '"4.2.3.1"'
         buildConfigField "int", "MINIMUM_WEBVIEW_VERSION", "55"
 
-        if (this.hasProperty("com.cliqz.lumen.vpnApiKey")) {
-            buildConfigField 'String', 'VPN_API_KEY', this.getProperty("com.cliqz.lumen.vpnApiKey")
+        def vpnApiKey = this.hasProperty("com.cliqz.lumen.vpnApiKey") ?
+                this.getProperty("com.cliqz.lumen.vpnApiKey") : System.getenv("vpnApiKey")
+        if (vpnApiKey != null) {
+            buildConfigField 'String', 'VPN_API_KEY', '"' + vpnApiKey + '"'
```

Comment:
`buildConfigField 'String', 'VPN_API_KEY', '"$(vpnApiKey)"'`

Reply:
@sharath-cliqz your suggestion will not work:
```groovy
buildConfigField 'String', 'VPN_API_KEY', "\"${vpnApiKey}\""
```

Reply:
Ahh. yes. Thank You @spacifici